File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/responses/FlightSearchWithBudgetResponse.java
package com.porkandlager.travelonbudget.wires.models.responses;

import com.google.gson.annotations.SerializedName;
import com.porkandlager.travelonbudget.wires.models.beans.FlightSearch;

import java.util.List;

/**
 * Created by tista on 11/12/16.
 */

public class FlightSearchWithBudgetResponse {

    private int status;
    private String message;

    @SerializedName("data")
    private List<FlightSearch> flights;

    public int getStatus() {
        return status;
    }

    public String getMessage() {
        return message;
    }

    public List<FlightSearch> getFlights() {
        return flights;
    }

}

File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/responses/FlightSearchWithBudgetResponse.kt
package com.porkandlager.travelonbudget.wires.models.responses

import com.google.gson.annotations.SerializedName
import com.porkandlager.travelonbudget.wires.models.beans.FlightSearch

/**
 * Created by tista on 11/12/16.
 */

class FlightSearchWithBudgetResponse {

    val status: Int = 0
    val message: String? = null

    @SerializedName("data")
    val flights: List<FlightSearch>? = null

}
